sdk/servicebus/azure-servicebus/azure/servicebus/_servicebus_receiver.py

```diff
@@ -399,7 +399,7 @@ def from_connection_string(
             raise ValueError("Subscription name is missing for the topic. Please specify subscription_name.")
         return cls(**constructor_args)
 
-    def receive_messages(self, max_message_count=None, max_wait_time=None):
+    def receive_messages(self, max_message_count=1, max_wait_time=None):
```

Comment:
You've brought great point on checking max_message_count > 0 if given, that's a valid scenario and we should probably do that like:

```python
If max_message_count is not None and max_message_count <= 0:
    raise ValueError("The max_message_count must be greater than 0").
```